draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -320,30 +320,34 @@ and can be sent in a trailer section.
 
 ## RateLimit-Remaining {#ratelimit-remaining-header}
 
-The `RateLimit-Remaining` response field indicates the remaining `quota-units` defined in {{request-quota}}
-associated to the client.
+The `RateLimit-Remaining` response field indicates the remaining `quota-units`
+defined in {{request-quota}} associated to the client.
 
-The header value is
+`RateLimit-Remaining` is a Item Structured Field
+{{!I-D.ietf-httpbis-header-structure}}. Its value MUST be an Integer (Section
+3.3.1 of {{I-D.ietf-httpbis-header-structure}}). Its ABNF is:
 
 ~~~
-   RateLimit-Remaining = quota-units
+   RateLimit-Remaining = sf-integer
 ~~~
 
-This header MUST NOT occur multiple times
-and can be sent in a trailer section.
+The value of `RateLimit-Remaining` MUST be non-negative.
 
-Clients MUST NOT assume that a positive `RateLimit-Remaining` value is
-a guarantee of being served.
-
-A low `RateLimit-Remaining` value is like a yellow traffic-light: the red light
-may arrive suddenly.
+`RateLimit-Remaining` MUST NOT occur multiple times. It MAY be sent in a trailer
+section.
 
-One example of `RateLimit-Remaining` use is below.
+For example:
 
 ~~~ example
    RateLimit-Remaining: 50
 ~~~
 
+Clients MUST NOT assume that a positive `RateLimit-Remaining` value is
+a guarantee of being served.
+
+A low `RateLimit-Remaining` value is like a yellow traffic-light: the red light
+may arrive suddenly.
+
```

Comment:
@ucarion Thanks for your PR. Just a work suggestion: every changes unrelated to structured headers in this PR distracts the reader from the focal point of this PR.

It is better to address editorial changes (eg. rewording, moving text around, ...) in specific PRs labeled with `editorial`: we can create one after we move all fields to SF.

Does it seem reasonable to you? Thanks again! R.